registration-service/src/main/java/com/example/registrationservice/constants/AppConstants.java

```diff
@@ -0,0 +1,10 @@
+package com.example.registrationservice.constants;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class AppConstants {
```

Comment:
Для утилитных классов отключают возможность их расширения.
```suggestion
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public final class AppConstants {
```